test/Microsoft.IdentityModel.Logging.Tests/LogHelperTests.cs

```diff
@@ -0,0 +1,404 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.IdentityModel.Logging.Tests
+{
+    using System;
+    using Microsoft.IdentityModel.Logging;
+    using Microsoft.IdentityModel.Tokens;
+    using Xunit;
+
+    // Leveraging Xunit collections: https://xunit.net/docs/running-tests-in-parallel#parallelism-in-test-frameworks
+    // Currently this does nothing but if we face issues with tests colliding trying to access ShowPII and LogCompleteSecurityArtifact
+    // this collection can be used to prevent race conditions.
+    [Collection("Relying on ShowPII and LogCompleteSecurityArtifact")]
```

Comment:
why wouldn't services also have a similar issue in production? I guess I'm not following what the issue is.

Reply:
It's about race conditions in unit tests.

Imagine two unit tests are executing in parallel and both modifying the static properties controlling how logs are generated. Those two unit tests will not be able to reliably predict the format and so will be flaky. Forcing them to run serially will make them more reliable. A better design still is not not rely on static variables but that's already pretty baked into Wilson unfortunately.